### PR TITLE
virtio-fs: fix global variables don't take effect

### PR DESCRIPTION
**Pull request by zhenyzha**
fix global variables don't take effect

ID: 1937608
Signed-off-by: Zhenyu Zhang <zhenyzha@redhat.com>

### CODE REVIEW

**Comment by zhenyzha** [comment]
Tested 50 times, the variable can be stable and effective.

 (1/50) repeat50.Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.aarch64.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_xfstest.with_local_source.with_cache.auto.arm64-pci: FAIL: The xfstest failed. (2297.79 s)
......
 (49/50) repeat49.Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.aarch64.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_xfstest.with_local_source.with_cache.auto.arm64-pci: FAIL: The xfstest failed. (2266.17 s)
 (50/50) repeat50.Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.aarch64.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_xfstest.with_local_source.with_cache.auto.arm64-pci: FAIL: The xfstest failed. (2297.79 s)

@YongxueHong @menli820 pls help me to review the patch if you are free, thanks.

**Comment by menli820** [comment]
LGTM

 (1/2) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_xfstest.with_local_source.with_cache.auto.q35: FAIL: The xfstest failed. (1659.80 s)
(2/2) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_xfstest.with_nfs_source.with_cache.auto.q35: FAIL: The xfstest failed. (2214.25 s)

File: qemu/tests/cfg/virtio_fs_share_data.cfg [hunk 1]
```diff
@@ -158,7 +158,9 @@
                     cmd_yum_install += 'make psmisc quota sed xfsdump xfsprogs libacl-devel libattr-devel libaio-devel '
                     cmd_yum_install += 'libuuid-devel xfsprogs-devel python3 sqlite'
                     cmd_make_xfs = 'cd /home/xfstests-dev/ && make && make install'
-                    cmd_setenv = 'echo -e 'TEST_DEV=${fs_target_fs1}\nTEST_DIR=${fs_dest_fs1}\nSCRATCH_DEV=${fs_target_fs2}\n'
+                    cmd_setenv = 'export TEST_DEV=myfs1 && export TEST_DIR=/mnt/myfs1 && export SCRATCH_DEV=myfs2 && '
+                    cmd_setenv += 'export SCRATCH_MNT=/mnt/myfs2 && export FSTYP=virtiofs && export FSX_AVOID="-E" && '
```
Comment (YongxueHong) [inline qemu/tests/cfg/virtio_fs_share_data.cfg:162]:
```suggestion
                    cmd_setenv = 'export TEST_DEV=${fs_target_fs1} && export TEST_DIR=${fs_target_fs1} && export SCRATCH_DEV=${fs_target_fs2} && '
                    cmd_setenv += 'export SCRATCH_MNT=${fs_dest_fs2} && export FSTYP=virtiofs && export FSX_AVOID="-E" && '
```

Comment (zhenyzha) [reply qemu/tests/cfg/virtio_fs_share_data.cfg]: thanks a lot, Corrected.

**Comment by zhenyzha** [comment]
(1/1) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.aarch64.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_xfstest.with_local_source.with_cache.auto.arm64-pci: FAIL: The xfstest failed. (2269.75 s)

Corrected,  Could you review it again if you are free, thanks.  @YongxueHong

**Review by YongxueHong — APPROVED** [review]
LGTM.

**Pull request merged**